Tests/ControllersTests/ResourcesControllerTest.swift

```diff
@@ -10,8 +10,10 @@ class ResourcesControllerTest: XCTestCase {
     let contents = ["file1": Array("this is a text file.".utf8),
                     "file2": Array("this is another text file.".utf8)]
 
+    ResourcesController.contents = [:]
+    ResourcesController.setData(contents: contents)
```

Comment:
Since I need data to be persisted across multiple requests, I moved them to static variables within the controllers that need them. This works in implementation, but my tests are now order dependent unless I clear the contents and re-set them to isolate each unit test.

Would this smell be considered to outweigh the benefits static variables give my program ?